modules/microadBidAdapter.js

```diff
@@ -115,6 +115,26 @@ export const spec = {
         params['aids'] = JSON.stringify(aidsParams)
       }
 
+      const pbadslot = bid.ortb2Imp?.ext?.data?.pbadslot;
+      const gpid = bid.ortb2Imp?.ext?.gpid || pbadslot;
+      if (gpid != null && gpid.length > 0) {
```

Comment:
@kida-yuga 
null, undefiend, 空文字列 全て以下で判定できないですかね？

```suggestion
      if (gpid) {
```

Reply:
こちらで修正しました。
[Microad Bid Adapter: To simple existing check](https://github.com/kida-yuga/Prebid.js/pull/2/commits/df9866209b330b6d938cc571513b62d13d8c7dfd)